### ISSUE TITLE
Issue on iPhone - Safari

### ISSUE THREAD

**Issue by clemorphy** [cubiq/add-to-homescreen#137]
I just tested your demo page (http://lab.cubiq.org/addtohome/demos/simple/) on Safari Mobile iPhone iOS 8.1 and iPhone iOS 7.0.4 and the message didn't show up.
It worked fine on Safari Mobile iPad iOS 7.1.2 and iPad Mini iOS 8.1

**Comment by kneeki** [comment]
@macsamiro localStorage capacity is set by the browser, usually it's capped at 5MB.

I'm having the issue with the message displaying however: The message will not display on iOS 8 (iPhone 5s) using any browser. I've found that I have exactly the same results if I am in "Toggle Device Mode" using the Chrome developer console.

```		addToHomescreen({
			skipFirstVisit: true,
			maxDisplayCount: 1,
			message:'Install the 360 Review on your home screen<br>Tap %icon and then <strong>&quot;Add to Home Screen&quot;.',
			lifespan: 3500
		});```

And my console.log:
``` function ath(options){_instance=_instance||new ath.Class(options);return _instance}
VM10681:3 {"lastDisplayTime":1420564583334,"returningVisitor":true,"displayCount":1,"optedout":false,"added":true}```

**Comment by cubiq** [comment]
`"returningVisitor":true,"displayCount":1`

with `maxDisplayCount: 1` means that the message won't be displayed again. can you try to increase the displaycount (just for testing)

**Comment by macsamiro** [comment]
@kneeki yes it's capped at 5MB..but per domain. Since cubiq says the script needs just few bytes i don't think the localstorage is the problem.

I tested it on a iPhone 6 iOS 8.1.1 & 8.1.2 and it works all good there...but on the iPhone 5 iOS 8.1.2 the message just does not want to appear.

**Comment by kneeki** [comment]
If I clear localStorage and upping (or simply removing) maxDisplayCount causes the message to display every time. If I refresh the page (without clearing) it never appears again as far as I can tell.

@macsamiro That's right. I was just trying to clear up any misconceptions since you mentioned your phone storage capacity. I may have just misread what you were talking about, so disregarding me may be a good idea. ;)

**Comment by dnizard** [comment]
Hello,

I'm having quiet the same issue on iOS 8.
However my problem is that the popup keeps displaying on and on, whatever options I've put.

the only way to make it disappear is to remove the "var auth" code.

Here is my configuration :

var ath = addToHomescreen({
    skipFirstVisit: false
    startDelay: 0,  
    lifespan: 0,   
    displayPace: 10080,   
    maxDisplayCount: 4,
    detectHomescreen: true
});

**Comment by mrmanishs** [comment]
Any updates on this issue? I cannot get it to show at all.

**Comment by j-lo** [comment]
Hey @cubiq, great script mate, thought I'd add to this as I'm having the same issue as well.
Like @macsamiro I can only get the pop-up to appear when the `privateModeOverride` is enabled, which then ignores all other settings and appears repeatedly, else it doesn't seem to work at all.
For reference, am testing on both an iPhone 5 running 9.2.1 and an iPad 4 running 9.2.1. Can't post the link as it's a private beta, can email it if you want.

**Comment by j-lo** [comment]
@cubiq, just to update and clarify, I can now get the pop-up to appear if I control it programatically, with something like the following code:

```
var ath = addToHomescreen({
				autostart: false
		});
		ath.show(true);
		ath.skipFirstVisit(false);
		ath.lifespan(2);
                ath.maxDisplayCount(2);
```

However while the skipFirstVisit setting works, others don't. Per @macsamiro's suggestion there seems to be some possible conflict with lifespan/displayPace/maxDisplayCount?
